ocs_ci/ocs/ui/base_ui.py

```diff
@@ -0,0 +1,116 @@
+import logging
+
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as ec
+from selenium.webdriver.common.by import By
+from webdriver_manager.chrome import ChromeDriverManager
+
+
+from ocs_ci.utility.utils import run_cmd, get_kubeadmin_password
+from ocs_ci.ocs.ui.views import login
+
+
+logger = logging.getLogger(__name__)
+
+
+class BaseUI:
+    """
+    Base Class for UI Tests
+
+    """
+
+    def __init__(self, driver):
+        self.driver = driver
+
+    def do_click(self, by_locator, type=By.XPATH, timeout=30):
+        wait = WebDriverWait(self.driver, timeout)
+        element = wait.until(ec.element_to_be_clickable((type, by_locator)))
+        element.click()
+
+    def do_send_keys(self, by_locator, text, type=By.XPATH, timeout=30):
+        wait = WebDriverWait(self.driver, timeout)
+        element = wait.until(ec.element_to_be_clickable((type, by_locator)))
+        element.send_keys(text)
+
+    def is_expanded(self, by_locator, type=By.XPATH, timeout=30):
+        wait = WebDriverWait(self.driver, timeout)
+        element = wait.until(ec.element_to_be_clickable((type, by_locator)))
+        return element.get_attribute("aria-expanded")
+
+    def choose_expanded_mode(self, mode, by_locator, type):
+        current_mode = self.is_expanded(by_locator=by_locator, type=type)
+        if mode != current_mode:
+            self.do_click(by_locator=by_locator, type=type)
+
+    def get_element_text(self, by_locator, type=By.XPATH, timeout=30):
+        wait = WebDriverWait(self.driver, timeout)
+        element = wait.until(ec.visibility_of_element_located((type, by_locator)))
+        return element.text
+
+    def get_title(self, title, type=By.XPATH, timeout=30):
+        wait = WebDriverWait(self.driver, timeout)
+        wait.until(ec.title_is(type, title))
+        return self.driver.title
+
+
+def login_ui(browser):
+    """
+    Login to OpenShift Console
+
+    Args:
+        browser(str): type of browser (chrome, firefox..)
+
+    return:
+        driver(Selenium WebDriver)
+
+    """
+    logger.info("Get URL of OCP console")
+    console_url = run_cmd(
+        "oc get consoles.config.openshift.io cluster -o"
+        "jsonpath='{.status.consoleURL}'"
+    )
+    logger.info("Get password of OCP console")
+    password = get_kubeadmin_password()
+    password = password.rstrip()
+    if browser == "chrome":
+        logger.info("chrome browser")
+        chrome_options = Options()
+        chrome_options.add_argument("--ignore-ssl-errors=yes")
+        chrome_options.add_argument("--ignore-certificate-errors")
+        # headless browsers are web browsers without a GUI
+        chrome_options.add_argument("--headless")
+        driver = webdriver.Chrome(
+            ChromeDriverManager().install(),
+            chrome_options=chrome_options,
+        )
```

Comment:
@dahorak @petr-balogh 
Need to install `chrome browser` (no chrome webdriver) on jenkins slave
https://gitlab.cee.redhat.com/ocs/ocs4-jenkins/-/blob/master/ansible/playbooks/jenkins-slave/playbook.yml

Reply:
```[jenkins@centos7-image-ocs-jenkins-slave-new ~]$ google-chrome --version
-bash: google-chrome: command not found
wget https://dl.google.com/linux/direct/google-chrome-stable_current_x86_64.rpm
sudo yum localinstall google-chrome-stable_current_x86_64.rpm
Complete!
[jenkins@centos7-image-ocs-jenkins-slave-new ~]$ google-chrome --version
Google Chrome 87.0.4280.141

```

Reply:
```
  File "test2.py", line 17, in <module>
    driver = webdriver.Chrome(ChromeDriverManager().install(),chrome_options=chrome_options)
  File "/home/jenkins/test/venv/lib64/python3.6/site-packages/webdriver_manager/chrome.py", line 28, in __init__
    chrome_type=chrome_type)
  File "/home/jenkins/test/venv/lib64/python3.6/site-packages/webdriver_manager/driver.py", line 54, in __init__
    self.browser_version = chrome_version(chrome_type)
  File "/home/jenkins/test/venv/lib64/python3.6/site-packages/webdriver_manager/utils.py", line 142, in chrome_version
    raise ValueError(f'Could not get version for Chrome with this command: {cmd}')
```

Reply:
@OdedViner could you use `chromium` since it is already installed on all the jenkins agents?

You just need to add one additional import:
```python
from webdriver_manager.utils import ChromeType
```
And update the `ChromeDriverManager()` usage like this:
```python
        driver = webdriver.Chrome(
            ChromeDriverManager(chrome_type=ChromeType.CHROMIUM).install(),
            chrome_options=chrome_options,
        )
```